server/statistics/hot_peer_cache.go

```diff
@@ -410,7 +410,7 @@ func (f *hotPeerCache) updateHotPeerStat(newItem, oldItem *HotPeerStat, bytes, k
 	newItem.rollingKeyRate = oldItem.rollingKeyRate
 
 	if newItem.justTransferLeader {
-		newItem.HotDegree = oldItem.HotDegree
+		newItem.HotDegree = 0
```

Comment:
Can you add more explanation about why we set it to 0? I'm not sure if it is reasonable.

Reply:
It will prevent the region that has just been transferred leader from being dispatched again, and because of the anti count, this region can still participate in the threshold calculation